.github/dependabot.yml

```diff
@@ -5,3 +5,21 @@ updates:
     schedule:
       interval: "monthly"
     open-pull-requests-limit: 10
+  - package-ecosystem: "docker"
+      directory: "/"
```

Comment:
```suggestion
    directory: "/"
```

Reply:
Sorry Chris, I didn't see this till after I fixed the error :disappointed: